Find a functionally equivalent code:android.widget.TimePicker.getCurrentHour:COMMENT
Method Modifier: public      
Comment:/**
 * @return The current hour in the range (0-23).
 */

@@ -1,10 +1,3 @@
 {
-    int currentHour = mHourSpinner.getValue();
-    if (is24HourView()) {
-        return currentHour;
-    } else if (mIsAm) {
-        return currentHour % HOURS_IN_HALF_DAY;
-    } else {
-        return (currentHour % HOURS_IN_HALF_DAY) + HOURS_IN_HALF_DAY;
-    }
+    return mDelegate.getCurrentHour();
 }

